content/en/monitors/guide/alert-on-no-change-in-value.md

```diff
@@ -1,6 +1,9 @@
 ---
-title: How can I configure a metric monitor to alert on NO change in value?
-kind: faq
+title: Configure a metric monitor to alert on NO change in value
+kind: guide
+aliases:
+  - monitors/faq/how-can-i-configure-a-metric-monitor-to-alert-on-no-change-in-value
+  - monitors/guide/how-can-i-configure-a-metric-monitor-to-alert-on-no-change-in-value
```

Comment:
This line is aliasing itself. Should be removed. Otherwise PR looks good 👍

Reply:
Good catch, removed!